socorro/external/es/crashstorage.py

```diff
@@ -522,92 +504,3 @@ def _submit_crash_to_elasticsearch(
                     exc_info=True,
                 )
                 raise
-
-
-class ESCrashStorageRedactedSave(ESCrashStorage):
-    required_config = Namespace()
-    required_config.namespace("es_redactor")
-    required_config.es_redactor.add_option(
-        name="redactor_class",
-        doc="the name of the class that implements a 'redact' method",
-        default="socorro.external.crashstorage_base.Redactor",
-        from_string_converter=class_converter,
-    )
-    required_config.es_redactor.add_option(
-        name="forbidden_keys",
-        doc="a list of keys not allowed in a redacted processed crash",
-        default=(
-            "json_dump, "
-            "upload_file_minidump_flash1.json_dump, "
-            "upload_file_minidump_flash2.json_dump, "
-            "upload_file_minidump_browser.json_dump"
-        ),
-        reference_value_from="resource.redactor",
-    )
-
-    required_config.namespace("raw_crash_es_redactor")
-    required_config.raw_crash_es_redactor.add_option(
-        name="redactor_class",
-        doc="the redactor class to use on the raw_crash",
-        default="socorro.external.es.crashstorage.RawCrashRedactor",
-        from_string_converter=class_converter,
-    )
-
-    def __init__(self, config, *args, **kwargs):
-        super().__init__(config, *args, **kwargs)
-        self.redactor = config.es_redactor.redactor_class(config.es_redactor)
-        self.raw_crash_redactor = config.raw_crash_es_redactor.redactor_class(
-            config.raw_crash_es_redactor
-        )
-
-    def prepare_crash_data(self, raw_crash, processed_crash):
-        self.raw_crash_redactor.redact(raw_crash)
-        self.redactor.redact(processed_crash)
-
-        super().prepare_crash_data(raw_crash, processed_crash)
-
-
-class ESCrashStorageRedactedJsonDump(ESCrashStorageRedactedSave):
-    """This class stores redacted crash reports into Elasticsearch, but instead
-    of removing the entire `json_dump`, it keeps only a subset of its keys.
-    """
-
-    required_config = Namespace()
-    required_config.add_option(
-        name="json_dump_allowlist_keys",
-        doc="keys of the json_dump field to keep in the processed crash",
-        default=[
-            "stackwalk_version",
-            "system_info",
-        ],
-        from_string_converter=list_converter,
```

Comment:
This bit was to re-add things that were removed in the superclass. Given we don't need the superclass anymore, we also don't need this class to undo some of the things the superclass did.